css/style1.css

```diff
@@ -7,7 +7,7 @@
 
 body {
   padding: 30px;
-  background-color: #7A0019;
+  background-color: #7a0019;
```

Comment:
Is there a reason for converting all the hex to lowercase? I don't know that I care a lot, and consistency is the most important thing. Historically hex is written in upper case, and I think most graphic design/photo editing systems still use upper case. Some searching suggests that there's no real consensus on whether CSS should be upper or lowercase. So any particular reason to switch to lowercase?